src/components/GameView/GameDialogs.vue

```diff
@@ -69,23 +69,13 @@ export default {
   },
   emits: ['clear-selection', 'handle-error'],
   computed: {
-    ...mapState({
-      playingFromDeck: ({ game }) => game.playingFromDeck,
-    }),
-
-    ...mapGetters([
-      'discarding',
-      'player',
-      'opponent',
-      'opponentPointTotal',
-      'opponentQueenCount',
-      'playerWins',
-    ]),
+    ...mapStores(useGameStore),
+    ...mapStores(useAuthStore),
```

Comment:
mapStores can input multiple stores
```suggestion
    ...mapStores(useGameStore, useAuthStore),
```